docs/package.json

```diff
@@ -28,7 +28,7 @@
     "@mdx-js/react": "^2.1.0",
     "@moxy/next-compile-node-modules": "^2.1.1",
     "@xstate/inspect": "^0.4.1",
-    "aws-amplify": "^5.0.1",
+    "aws-amplify": "^5.0.16",
```

Comment:
Removing `skipLibCheck` caused `react-core` to have linting failures from a type issue in `@aws-amplify/storage@5.0.1` which was fixed in https://github.com/aws-amplify/amplify-js/pull/10696.

Upgrading `aws-amplify` here as it is the version that gets hoisted in to the top level _node_modules_.